Source/WebGPU/WGSL/TypeDeclarations.rb

```diff
@@ -8,13 +8,22 @@
 }
 
 operator :*, {
+    [T < Number, N].(Vector[T, N], T) => Vector[T, N],
```

Comment:
nit: I wonder if we should add comments to document what these overloads are for? Eg:

```
# Vector scaling